janrain_capture.install

```diff
@@ -118,3 +118,65 @@ function janrain_capture_uninstall() {
       ->condition('name', "janrain_capture_share%", 'LIKE'))
     ->execute();
 }
+
+/**
+ * Update users.
+ */
+function janrain_capture_update_7000(&$sandbox) {
```

Comment:
`hook_update_7000` usually is used for upgrade from D6 module version.
Please rename to `janrain_capture_update_7001`.